common/app/views/fragments/head.scala.html

```diff
@@ -10,7 +10,9 @@
 @* stylesheet <link>s - get the stylesheets downloading ASAP *@
 @fragments.stylesheets(projectName, getContent(page).exists(_.tags.isCrossword))
 
-<link rel="prefetch" href="@Static("javascripts/app.js")">
+@if(!play.Play.isDev()) {
+    <link rel="prefetch" href="@Static("javascripts/app.js")">
+}
```

Comment:
it triggers one of these

<img width="719" alt="screen shot 2016-01-27 at 17 04 15" src="https://cloud.githubusercontent.com/assets/867233/12621339/086f1c16-c518-11e5-9152-21a1503dd47d.png">